doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -3062,16 +3013,6 @@ service evmserverd start
 ------
 +
 
-[IMPORTANT]
-===========
-*Requisites for Replication*
-
-If you are using replication, and you have changed the password on the *Master* database, you will also need to update the password used for the *Replication Worker* on the subordinate regions. To do this:
-
-. Disable the *Database Synchronization* Role.
-. Update the *Replication Worker*'s password.
-. Restart the *Database Synchronization* Role.
-===========
```

Comment:
This may be worth altering to fit the new replication architecture.

Now, if you change the password on the *remote* database, you will need to change the password in the subscription.